code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -392,18 +392,12 @@ GLOBAL_LIST_INIT(brass_recipes, list ( \
 	throw_range = 3
 	turf_type = /turf/open/floor/clockwork
 	novariants = FALSE
-	grind_results = list("iron" = 5, "teslium" = 15)
+	grind_results = list("brass" = 20)
 
 /obj/item/stack/tile/brass/narsie_act()
 	new /obj/item/stack/sheet/runed_metal(loc, amount)
 	qdel(src)
 
-/obj/item/stack/tile/brass/attack_self(mob/living/user)
```

Comment:
don't

Reply:
do, emergent gameplay

Reply:
do, it's cool to build stuff with cultist's materials once you arrest them

also it's a kind of important part to the PR

Reply:
don't because you also have to change runic metal

Reply:
Not really, there's no easy source of runic metal. Best one is probably lavaland and almost no one does that.

Reply:
Also runic metal is very different, functionally. Runic metal makes functional structures which are useless as a non-cultist, brass makes clock-themed walls and windows and skewers/steam vents( which are 100% usable as a noncult)